test/dummy/db/schema.rb

```diff
@@ -196,7 +181,6 @@
     t.string "acc_status", default: "pending"
     t.integer "acc_status_author_id"
     t.datetime "acc_status_date"
-    t.boolean "riding_bus", default: false
```

Comment:
I'm pretty sure we need a migration to drop this column? Both BrickHack and WiCHacks have this in their db/schema.rb definitions